pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -600,7 +601,7 @@ def to(self, *args, **kwargs) -> 'ResultCollection':
         def to_(item: Union[torch.Tensor, Metric], *args: Any, **kwargs: Any) -> Union[torch.Tensor, Metric]:
             return item.to(*args, **kwargs)
```

Comment:
Do you want to replace this with `move_data_to_device`?

Reply:
Done !